src/services/tags-service.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file Implements the service for tuits
```

Comment:
Just a documentation typo here.  Should be service for tags instead of service for tuits. Nbd.